packages/app/src/components/pages/Home/index.js

```diff
@@ -1,12 +1,56 @@
-import React from 'react';
+import React, {useEffect, useContext} from 'react';
 import SearchBar from '../../SearchBar';
+import NewFilesToReview from './NewFilesToReview';
+import states from 'us-state-codes';
+import nominationsAPI from '../../../utils/API/nominationsAPI';
+import { NominationsDataContext } from '../../../utils/context/NominationsContext';
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { fab } from '@fortawesome/free-brands-svg-icons'
+import { faChevronCircleDown, faChevronCircleUp, faFileImage, faEllipsisV } from '@fortawesome/free-solid-svg-icons'
+library.add(fab, faChevronCircleDown, faChevronCircleUp, faFileImage, faEllipsisV)
+
+const Home = () => {
+  const [NominationsData, setNominationsData] = useContext(
+    NominationsDataContext
+  );
+
+// TODO figure out way to garuantee global state is set before things render
```

Comment:
could use a conditional render or could set a loading state if want to wait for nominations call to complete
https://andrewmmc.com/blog/2020/handle-fetch-state-in-react-hooks